app-games/hmcl/autobuild/build

```diff
@@ -1,9 +1,15 @@
+abinfo "Building hmcl.jar..."
+export VERSION_TYPE=stable
+export BUILD_NUMBER=$(echo $PKGVER | cut -d '.' -f 3)
+export MICROSOFT_AUTH_ID=81a207c0-a53c-46a3-be07-57d2b28c1643
+export CURSEFORGE_API_KEY="$2a$10$gWtfk4i6mQvxSLhtMdvBfuM1E46vEbmYV4w4sHSJBX6c3hO6WCDWG"
+./gradlew build
+
 abinfo "Installing hmcl.jar..."
 mkdir -pv "${PKGDIR}/usr/share/java/hmcl"
-cp -Lv "${SRCDIR}"/*.jar "${PKGDIR}/usr/share/java/hmcl/hmcl.jar"
+ls "${SRCDIR}"/HMCL/build/libs/ | grep ".jar$"
+cp -Lv "${SRCDIR}"/HMCL/build/libs/HMCL*.jar "${PKGDIR}/usr/share/java/hmcl/hmcl.jar"
 
-abinfo "Extracting desktop icon from jar..."
+abinfo "Extracting desktop icon..."
 mkdir -pv "${PKGDIR}"/usr/share/pixmaps
-unzip -p "${PKGDIR}/usr/share/java/hmcl/hmcl.jar" \
-      assets/img/icon\@8x.png \
-      > "${PKGDIR}"/usr/share/pixmaps/hmcl.png
+cp "${SRCDIR}"/HMCL/image/hmcl.png "${PKGDIR}"/usr/share/pixmaps/hmcl.png
```

Comment:
Merge the above two commands as follows:

```bash
install -Dvm644 "$SRCDIR"/HCML/image/hmcl.png \
    "$PKGDIR"/usr/share/pixmaps/hmcl.png
```

Reply:
`"$SRCDIR"/HCML/...` should be `"$SRCDIR"/HMCL/...`  x